src/handlebars/index.js

```diff
@@ -13,7 +13,8 @@ module.exports = function (conf) {
 
 	return handlebars(app, {
 		partialsDir: [
-			directory + (options.viewsDirectory || '/views') + '/partials'
+			directory + (options.viewsDirectory || '/views') + '/partials',
+			directory + ('/node_modules/@financial-times')
```

Comment:
I wonder if this should look in a subfolder to reduce cost at startup? Maybe `/views`? Might be good to have a convention, what do you think?